<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectlevel</class>
 <widget class="QWidget" name="selectlevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/setting/dialogbox.png);</string>
  </property>
  <widget class="QPushButton" name="level1B">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/setting/nandu.png);
font: 22pt &quot;华文行楷&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>简单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level2B">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/setting/nandu.png);
font: 22pt &quot;华文行楷&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>普通</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level3B">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/setting/nandu.png);
font: 22pt &quot;华文行楷&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>困难</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/setting/Ok.png);
font: 20pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>选择模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="DoneB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/setting/enter.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
